Automattic/newspack-newsletters
# feat(cli): add command to synchronize premium lists

### All Submissions:






### Changes proposed in this Pull Request:



Adds a CLI command to synchronize premium newsletter subscribers. 

### How to test the changes in this Pull Request:

1. Ensure you have a premium newsletter list:
    1. Create a membership plan with "Restrict Content" -> choose "Subscription Lists" -> pick a list
3. Create two memberships for two different users, with this plan 
4. Cancel one of the memberships 
5. In the ESP, break things: unsubscribe the user with the active membership, and subscribe the user with the inactive membership
6. Run `wp newspack-newsletters sync-membership-tied-subscribers --live --verbose`, and observe it realigns the status in the ESP with the memberships status 

### Other information:




---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208660941160628
